.github/actions/bump-version/index.js

```diff
@@ -17,35 +16,16 @@ const run = async () => {
 
     core.info('Set variables');
     const version = latestRelease.data.tag_name;
```

Comment:
The tag name includes the version with the 'v' prefix, so the PR updates the package.json and the manifest.json as: 
```json
"version": "v1.1.1"
```

 I'm not sure if that is an issue but perhaps we could split by "v" and take the second item of the result.

 Note: The .zip artifact is build correctly wihtout the prefix.

Reply:
Done